File: Composicion_y_Agregacion/Ejercicio3/Java/Parte.java
package Composicion_y_Agregacion.Ejercicio3.Java;


public class Parte { 
        private String nombre;
        double peso;

    public Parte(String nombre, double peso) { 
        this.nombre = nombre; 
        this.peso = peso;
        }
    void mostrar_info() {
        System.out.println("\nNombre parte: + Nicol");
        System.out.println("Peso: " + peso + "kg");
    }

    String getNombre() { return nombre; } 
    double getPeso() { return peso; }
}

File: Composicion_y_Agregacion/Ejercicio3/Python/Parte.py

class Parte:
    def __init__(self, nombre, peso):
        self.nombre = nombre
        self.peso = peso   
    def mostrar_info(self):
        print(f"\nNombre parte: {self.nombre}")
        print(f"Peso: {self.peso} kg")
    def get_nombre(self):  
        return self.nombre
    def get_peso(self):  
        return self.peso
